lib/approvals/writers/custom_writer.rb

```diff
@@ -0,0 +1,13 @@
+module Approvals
```

Comment:
Alternatively, what if we allowed something like `verify, format: custom, "CustomWriter" do...end`? Then this file would be irrelevant, and users could define multiple custom strategies and use whichever made sense in that context.